### DISCUSSION TITLE
Order history

### DISCUSSION THREAD

**Discussion by megataps** [vendurehq/vendure#825] [General]
When using Vendure Framework as an admin backend for mobile app. I don't know how to get order history for logged in customer to see all the order that he/she made?

Does anyone know how to do this? Is it already have in the core framework or We have to implement this feature?

**Comment by michaelbromley** [comment]
Hi,

The `Customer` type in the GraphQL API has an `orders` field, which exposes a list of all orders by that customer: https://www.vendure.io/docs/graphql-api/admin/object-types/#customer

**Reply by megataps** [reply]
Thanks @michaelbromley
